src/analyzer/protocol/finger/finger.spicy

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 by the Zeek Project. See LICENSE for details.
+
+module Finger;
+
+import zeek;
```

Comment:
Valid point. I went for a single file because these analyzers will be shipping inside the Zeek tree, but I can see going the other way too.

Another thought: the most common use-case for Zeek-specific logic is calling the DPD confirm/reject functions. We could do that differently, so that it doesn't need any Zeek-specific code. For one, calls to `zeek::reject_protocol` might often be unnecessary on the first place because the plugin does them anyways when an error bubbles up; so could just remove. I'll just do that for both syslog and finger.

For `confirm_protocol` we could either extend EVT files to allow doing it from there (say, `on Foo::%done -> confirm`); or we could give Spicy some generic function (`spicy::confirm_protocol`) that host applications can hook into. Then the Zeek plugin would tie that to DPD's confirmation, and other host applications could just leave it a no-op if they don't have anything comparable.

Reply:
> For one, calls to `zeek::reject_protocol` might often be unnecessary on the first place because the plugin does them anyways when an error bubbles up; so could just remove. I'll just do that for both syslog and finger.

Turns out the plugin already raises violations automatically for unhandled parse errors, so safe to just skip.

> we could give Spicy some generic function (`spicy::confirm_protocol`) that host applications can hook into. Then the Zeek plugin would tie that to DPD's confirmation, and other host applications could just leave it a no-op if they don't have anything comparable.

This seems generally useful, so I'm going that route.